tests/deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
All three of these scripts are quite tightly coupled (binary locations, logic, actual things deployed, etc), for easier navigability and maintainability I would suggest to merge them into one single scripts called tests/e2e.sh

Reply:
The idea right now is, that one can easily run `deploy.sh` once, if they already have a kind cluster running. Then run the tests multiple times, without running all of it from the beginning to the end.

Reply:
Ahh interesting. I see how that would be useful. Hmm right now the scripts are so tightly coupled that it seems a bit disingenuous to put them in separate scripts. It would be great for them to be separate functions in the same script. We can definitely merge as is; it’s not a blocker for me. I’ll can make a followup PR to show what I mean

Reply:
Let's do it in a follow up and discuss how we might be able to have one script and maybe flags to run individual functions only.